mkdocs.yml

```
@@ -67,6 +67,9 @@ markdown_extensions:
   - pymdownx.blocks.caption
   - footnotes
   - mdx_truly_sane_lists
+  - redirects:
+      redirect_maps:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)